uber/site_sections/jobs.py

```diff
@@ -36,7 +36,8 @@ def update_counts(job, counts):
 
 @all_renderable(c.PEOPLE)
 class Root:
-    def index(self, session, location=None, message=''):
+
+    def index(self, session, location=None, message='', time=None):
```

Comment:
Silly question: you just added an option ``time`` attribute but aren't actually using it anywhere in this page handler.  Did you add that by mistake?